### DISCUSSION TITLE
"grey bar" in horizontal stack

### DISCUSSION THREAD

**Discussion by JVSDR92** [Clooos/Bubble-Card#331] [Questions about config, custom styles and templates] [answered]
Hi everyone,

I got a problem with my horizontal stack, i have a grey bar over the full width of my screen and i can't figure out how to fix this. I tried mutiple things but at this moment I have no clue anymore. 
Does somebody recognize this problem and know how to fix it? That would be very helpfull!

![Schermafbeelding 2023-12-14 214107](https://github.com/Clooos/Bubble-Card/assets/144735045/82806151-4cfd-41b4-8c84-0420b76b35c6)
![Schermafbeelding 2023-12-14 214132](https://github.com/Clooos/Bubble-Card/assets/144735045/ffc9bd40-ce6c-4e43-bfea-a3610f1f197a)

**Comment by Raabjerg** [comment, marked as answer]
Hi.
You can get it to disappear by adding this code in the Bubble card configuration/Horizontal buttons stack “Show code editor”
styles: |
  .horizontal-buttons-stack::before {
    background: none !important;
  }
![Billede1](https://github.com/Clooos/Bubble-Card/assets/125222045/ecef4dd7-94d2-45b2-b913-9695a2a7a9b4)
![Billede2](https://github.com/Clooos/Bubble-Card/assets/125222045/63c1427e-2cc0-4adf-aec2-6d347e6a240e)
